vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -233,3 +233,70 @@ def dumbbell(data_frame: pd.DataFrame, x: str, y: str, color: str) -> go.Figure:
     # Increase size of dots
     fig.update_traces(marker_size=12)
     return fig
+
+
+@capture("graph")
+def diverging_stacked_bar(
+    data_frame,
+    y: str,
+    category_pos: List[str],
+    category_neg: List[str],
+    color_discrete_map: Optional[Dict[str, str]] = None,
+) -> go.Figure:
+    """Creates a horizontal diverging stacked bar chart (with positive and negative values only) using Plotly's go.Bar.
+
+    This type of chart is a variant of the standard stacked bar chart, with bars aligned on a central baseline to
+    show both positive and negative values. Each bar is segmented to represent different categories.
```

Comment:
I decided to go for the diverging stacked bar chart (without a neutral group). Enabling the neutral group would be a lot more complex and in general that chart with the neutral group is not really a recommended one given the difficulties to interpret it: https://blog.datawrapper.de/divergingbars/

In general, this is one of those charts which look very fancy, but might not be a good recommended chart. I recommend reading through the article above :)

Reply:
Very good article indeed! Maybe at some point in future we could have one with the neutral group separated off like they suggest, but definitely seems like there's a strong case to be made for just not doing this kind of plot at all 👍